include/groonga/groonga.h

```diff
@@ -1062,6 +1062,28 @@ grn_column_rename(grn_ctx *ctx,
                   const char *name,
                   unsigned int name_size);
 
+/**
+ * \brief Frees a temporary object from memory.
+ *
+ *        This function frees the specified object (`obj`) from memory.
+ *        If `obj` includes other associated objects, it recursively frees
+ *        those as well.
```

Comment:
```suggestion
 * \brief Close an object.
 *
 *        This function frees all resources used by the specified obje (`obj`) from
 *        memory. All resources include other associated objects.
 *
 *        In general, you must close temporary objects explicitly. You don't need to
 *        close persistent objects explicitly because you can close persistent objects
 *        implicitly by closing a DB object.
 *
 *        You can use \ref grn_obj_unlink instead. It closes temporary objects but
 *        does nothing for most persistent objects. It's useful for normal use cases.
```